### DISCUSSION TITLE
PR the bugfixes into Scratch's repositories

### DISCUSSION THREAD

**Discussion by just-a-hriday** [ScratchAddons/ScratchAddons#3450] [Ideas] [answered]
Maybe pull requests should be created in Scratch's [repositories](https://github.com/LLK) for all the addons that simply fix bugs?

**Comment by CST1229** [comment, marked as answer]
Problem is, Scratch [doesn't accept PRs for issues that don't have a priority label/the help wanted label](https://github.com/LLK/scratch-gui/blob/9f2b4da58f7266447a69f306b923e3ba19bda81b/.github/CONTRIBUTING.md).
And (for example)...
![nooo](https://user-images.githubusercontent.com/68464103/132704514-4498b367-2ca7-4968-bb3c-d3a72cafeb88.png)
(from LLK/scratch-gui#1307)

**Reply by TheColaber** [reply]
~~I think u read the post wrong~~ i think i did lol

**Reply by just-a-hriday** [reply]
> I think u read the post wrong

Nope, they read it correctly. Well, I guess we can't PR those bugfixes, then.
